src/Uno.UI.RemoteControl/HotReload/ClientHotReloadProcessor.Xaml.cs

```diff
@@ -38,6 +39,27 @@ namespace Uno.UI.RemoteControl.HotReload
 	partial class ClientHotReloadProcessor
 	{
 		private string? _lastUpdatedFilePath;
+		private bool _supportsXamlReader;
+
+		private void InitializeXamlReader()
+		{
+			var targetFramework = GetMSBuildProperty("TargetFramework");
+			var buildingInsideVisualStudio = GetMSBuildProperty("BuildingInsideVisualStudio").Equals("true", StringComparison.OrdinalIgnoreCase);
```

Comment:
not sure if it is on purpose but the variable `buildingInsideVisualStudio` is not being used

Reply:
Indeed it's not I'll remove it in a separate PR. Thanks!